Docs: Relabel "Single version documentation" documentation from feature to explanation (Diátaxis) (#9835)
readthedocs/readthedocs.org: docs/user/single_version.rst

Single version documentation
----------------------------

> *Single version* documentation lets you serve your documentation direct from ``https://docs.example.com`` or ``https://example.readthedocs.io``.

Reviewer: ```suggestion
*Single version* documentation lets you serve your documentation at the top level of your domain,
for example ``https://docs.example.com/`` or ``https://example.readthedocs.io/``.
```

> By default, all documentation served by Read the Docs has a root of ``/<language>/<version>/``.

Reviewer: ```suggestion
By default, all documentation served by Read the Docs has a standard path of ``/<language>/<version>/``.
```

> But, if you enable the *single version* option for a project, its documentation will instead be served at ``/``.

Reviewer: ```suggestion
Documentation can be served at ``/`` if you enable the *single version* option for a project. 
```

Clearer I think.

.. warning:: This means you can't have translations or multiple versions for your documentation.

Having a single version of a documentation project can be considered the better choice
in cases where there should only always exist one unambiguous copy of your project.

> For example, a research project may wish to *only* expose readers to their latest list of publications and research data.

Reviewer: ```suggestion
For example, a research project may wish to *only* expose readers to their latest list of publications and research data.
Similarly, a :abbr:`SaaS (Software as a Service)` application might only ever have one version live. 
```

Author: We know that example very well :100:
